tests/e2e/test_actions.py

```diff
@@ -49,6 +49,7 @@ def test_actions_sanity(update_environment):
     cmd = utils.Command(
         rulebook=rulebook,
         inventory=inventory,
+        execution_strategy="parallel",
```

Comment:
Why this is needed? If sequential is the default strategy doesn't seem a good idea to run the test with parallel. I guess it is because we have fails, don't? In that case I prefer to update the affected tests. 

Also we would need more tests to ensure both strategies. I will create a ticket for it.

Reply:
@Alex-Izquierdo That particular test is written with parallel execution in mind. So the only way that test can pass is if we set the execution strategy to parallel